tests/test_brainreg_napari.py

```diff
@@ -167,6 +170,9 @@ def check_orientation_output(viewer, brain_layer, atlas):
     assert viewer.layers[3].data.shape == (atlas.shape[0], atlas.shape[1])
 
 
+@pytest.mark.xfail(
+    reason="See https://github.com/brainglobe/cellfinder/issues/443"
```

Comment:
```suggestion
    reason="See https://github.com/brainglobe/cellfinder/issues/443",
    raises=AssertionError
```
as above.

Reply:
Good call, should've just copied it from `cellfinder`!